addons/im_livechat/static/src/legacy/components/attachment_delete_confirm/attachment_delete_confirm.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates xml:space="preserve">
-    <t t-name="mail.AttachmentDeleteConfirmView" owl="1">
+    <t t-name="im_livechat.AttachmentDeleteConfirmView" owl="1">
```

Comment:
I would suggest to have `.legacy` in name of those, but I feel they will all be removed at a next step (was a bit useless to move them in the first place).